code/__DEFINES/loadout.dm

```diff
@@ -200,7 +200,7 @@ GLOBAL_LIST_INIT(loadout_linked_vendor, list(
 		/obj/machinery/vending/marineFood,
 		/obj/machinery/vending/MarineMed,
 		/obj/machinery/vending/cigarette,
-		/obj/machinery/vending/cigarette/colony,
```

Comment:
didn't make any sense, how would the loadout vendor connected to a colony vendor

Reply:
cuz i stuck em all together in lazyness. good change